packages-common/nanoflow-commons/src/client/Refresh.ts

```diff
@@ -0,0 +1,39 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+
+// BEGIN EXTRA CODE
+// END EXTRA CODE
+
+/**
+ * @param {string} objectGuid - Pass guid in order to trigger object subscriptions
+ * @param {string} attributeName - Pass attribute along with GUID in order to trigger attribute subscriptions
+ * @param {string} entityName - Pass entityName in order to trigger entity subscriptions
+ * @returns {Promise.<void>}
+ */
+export async function Refresh(entityObject?: mendix.lib.MxObject, entityName?: string) {
+    // BEGIN USER CODE
+    let obj = {};
+
+    if (entityName) {
+        obj = {
+            entity: entityName
+        };
+    } else if (entityObject) {
+        obj = {
+            guid: entityObject.getGuid()
+        };
+    }
+
+    return new Promise(resolve => {
+        mx.data.update({
+            ...obj,
+            ...{ callback: () => resolve(true) }
+        } as any);
```

Comment:
- i believe this `any` is avoidable and should be avoided
- you are missing the `error`